src/highdicom/content.py

```diff
@@ -936,3 +1067,536 @@ def __init__(
+                    )
+                self.WindowWidth = format_number_as_ds(window_width)
+        if window_explanation is not None:
+            if window_center is None:
+                raise TypeError(
+                    'Providing "window_explanation" is invalid if '
+                    '"window_center" is not provided.'
+                )
+            if isinstance(window_explanation, str):
+                if window_is_sequence:
+                    raise TypeError(
+                        'Length of "window_explanation" must match length of '
+                        '"window_center".'
+                    )
+                _check_long_string(window_explanation)
+            elif isinstance(window_explanation, Sequence):
+                if (
+                    not window_is_sequence or
+                    (len(window_explanation) != len(window_center))
+                ):
+                    raise ValueError(
+                        'Length of "window_explanation" must match length of '
+                        '"window_center".'
+                    )
+                if len(window_explanation) == 0:
+                    raise TypeError(
+                        'Argument "window_explanation" must not be an empty '
+                        'sequence.'
+                    )
+                for exp in window_explanation:
+                    _check_long_string(exp)
+            self.WindowCenterWidthExplanation = window_explanation
+        if voi_lut_function is not None:
+            if window_center is None:
+                raise TypeError(
+                    'Providing "voi_lut_function" is invalid if '
+                    '"window_center" is not provided.'
+                )
+            self.VOILUTFunction = VOILUTFunctionValues(voi_lut_function).value
+
+        if voi_luts is not None:
+            if len(voi_luts) == 0:
+                raise ValueError('"voi_luts" should not be empty.')
+            for lut in voi_luts:
+                if not isinstance(lut, LUT):
+                    raise TypeError(
+                        'Items of "voi_luts" should be of type highdicom.LUT.'
+                    )
+            self.VOILUTSequence = list(voi_luts)
+        else:
+            if window_center is None:
+                raise TypeError(
+                    'At least one of "window_center" or "voi_luts" should be '
+                    'provided.'
+                )
+
+
+class PaletteColorLookupTable(Dataset):
+
+    """Dataset describing a palette color lookup table."""
+
+    def __init__(
+        self,
+        red_palette_color_lut_data: np.ndarray,
+        green_palette_color_lut_data: np.ndarray,
+        blue_palette_color_lut_data: np.ndarray,
+        red_first_mapped_value: int,
+        green_first_mapped_value: int,
+        blue_first_mapped_value: int,
+        palette_color_lut_uid: Union[UID, str, None] = None
+    ):
+        """Add attributes from the Palette Color Lookup Table module.
```

Comment:
For now, I would keep DICOM modules out of the library as much as possible.

```suggestion
        """
```